website/content/cloud-native-sustainability-week/_index.md

```diff
@@ -0,0 +1,90 @@
+These meetups can be organized by anyone interested in supporting the effort. If you would like to contribute, check out the [Contact and Resources](#contact-and-resources) section below.
+
+There will be two types of events organized during the Cloud Native Sustainability Week:
+1. **Local in-person events**: Meetups organized worldwide during the second week of October 2023 by [CNCF community groups](https://community.cncf.io/chapters/) about cloud native sustainability.
+2. **Virtual Event**: A virtual meetup organized by the CNCF TAG ENV.
+
+Please take a look at this [document](https://docs.google.com/document/d/1s28lqqc3IBAMw4T7n13zfyAuUEEc3I4_xhPxCLA4_dk/edit?usp=sharing) for a more detailed overview.
+
+## Meetup Locations
+
+*Note that this list is changing, we will publish a complete overview a few weeks for the event! Take a look at this [document](https://docs.google.com/document/d/1s28lqqc3IBAMw4T7n13zfyAuUEEc3I4_xhPxCLA4_dk/edit#heading=h.19phjl5j6fdw) to check the latest status, and persons to contact.*
+
+
+
+| **Nr** | **Location** | **Date** | **Event Link** | **Name** |
+|---|---|---|---|---|
+| 1 | Australia, Sydney | *tbd* |  *tbd* | Adam Gardner, Prateek Nayak |
+| 2 | Brasil, São Paulo | *tbd* |  *tbd* | Carol Valencia |
+| 3 | Canada, Toronto |  *tbd* |  *tbd* | Ayrat Khayrettdinov |
+| 4 | Denmark, Aarhus |  *tbd* |  *tbd* | Kasper Nissen |
+| 5 | England, London |  *tbd* |  *tbd* | Kunal Kushwaha |
+| 6 | Germany, Hamburg | *tbd* | *tbd* | Leonard Pahlke |
+| 7 | Germany, Karlsruhe | *tbd* | *tbd* | Aydin Mir Mohammadi |
+| 8 | Germany, Munich | *tbd* | *tbd* | Antonio Di Turi, Max Körbächer |
+| 9 | Germany, Berlin | *tbd* | *tbd* | Adam Roe |
+| 10 | Guatemala | *tbd* | *tbd* | Sergio Mendez |
+| 11 | India, Bangalore | *tbd* | *tbd* | Nancy Chauhan, Divya Mohan |
+| 12 | India, Chennai | *tbd* | *tbd* | Vinothini B, Manikandan, Vijayabharathi |
+| 13 | Italy, Milan | *tbd* | *tbd* | Michel Murabito |
+| 14 | Korea, Seoul | *tbd* | *tbd* | Seokho Son, Hoon Jo |
+| 15 | Latin America | *tbd* | *tbd* | Angel Ramirez |
+| 16 | Luxembourg, Luxembourg | *tbd* | *tbd* | Stéphane Este-Gracias |
+| 17 | Mexico, Guadalajara | *tbd* | *tbd* | Rodolfo Martínez Vega |
+| 18 | Netherlands, Amsterdam | *tbd* | *tbd* | Alessandro Vozza |
+| 19 | Norway, Oslo | *tbd* | *tbd* | Kristina Devochko |
+| 20 | Spain, Barcelona | *tbd* | *tbd* | Niki Manoledaki |
+| 21 | Switzerland, Bern | *tbd* | *tbd* | Johann Gyger |
+| 22 | Taiwan, Taipei | *tbd* | *tbd* | Phil Huang |
+| 23 | USA, Chicago | *tbd* | *tbd* | Josh Gavant |
+| 24 | USA, Oakland | *tbd* | *tbd* | Rynn Mancuso |
+| 25 | USA, Seattle | *tbd* | *tbd* | Yiming Peng|
```

Comment:
One quick question, if we want to contribute with meet up, how should I update my contact info here?
as information below?
| 26 | China, Beijing | *tbd* | *tbd* | Sam Yuan(github id: SamYuan1990) |

Reply:
@SamYuan1990  you'd likely just want to add in a PR with the contact info added.  @leonardpahlke  is this what you imagined?

Reply:
Yes, you can just open a PR and update the details.

Reply:
We will reach out to all in a few weeks and update info's as a batch. But.. Feel free to open PRs at anytime to change the update the records! thanks @SamYuan1990 :)

Reply:
https://github.com/cncf/tag-env-sustainability/pull/147 here you are :-) please advise.